### DISCUSSION TITLE
What is the importance to implement Cryptography Engineering?

### DISCUSSION THREAD

**Discussion by PurpleMonkey729** [PurpleMonkey729/Cryptography_Overview#3] [Q&A] [answered]
What is the importance to implement Cryptography Engineering?

**Comment by toptal0212** [comment, marked as answer]
Individuals and organizations use cryptography on a daily basis to protect their privacy and keep their conversations and data confidential. Cryptography ensures confidentiality by encrypting sent messages using an algorithm with a key only known to the sender and recipient.

https://www.fortinet.com/resources/cyberglossary/what-is-cryptography#:~:text=The%20Importance%20of%20Cryptography&text=Individuals%20and%20organizations%20use%20cryptography,to%20the%20sender%20and%20recipient.

**Comment by PurpleMonkey729** [comment]
Thx for your reply.
